openfl-tutorials/interactive_api/PyTorch_Histology_FedCurv/envoy/populate_envoys.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+for i in {1..8}
+do
+    mkdir $i
+    echo "shard_descriptor:
+    template: histology_shard_descriptor.HistologyShardDescriptor
+    params:
+        data_folder: histology_data
+        rank_worldsize: $i,8
+" > $i/envoy_config.yaml
+    echo "fx envoy start -n env_$i --disable-tls --envoy-config-path envoy_config.yaml -dh nnlicv431.inn.intel.com -dp 50053" > $i/start_envoy.sh
```

Comment:
Can we parametrize `-dh nnlicv431.inn.intel.com -dp 50053` with variables, and maybe insert everywhere `localhost`?